src/shared/components/palette/text-area/write-text-area-field.component.ts

```diff
@@ -16,7 +16,11 @@ export class WriteTextAreaFieldComponent extends AbstractFieldWriteComponent imp
   }
 
   ngOnInit() {
-    this.textareaControl = this.registerControl(new FormControl(this.caseField.value)) as FormControl;
+    const notEmpty = this.caseField.value !== null && this.caseField.value !== undefined;
+    this.textareaControl = this.registerControl(new FormControl(notEmpty ? this.caseField.value : null)) as FormControl;
+    if (this.textareaControl.disabled) {
```

Comment:
Same as above:
This takes no account of why the control has been disabled. The conditional show/hide logic will hide and disable fields that shouldn't be shown so that the data from those hidden fields doesn't get submitted. If you've added code to simply enable anything that's disabled - which is what this does - there's every chance the submitted data will be invalid or, at least, that you'll be submitting data for fields that are hidden and shouldn't be submitted.